### PR TITLE
Fix release note generation

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-14261
affects: none
release note: none

### CODE REVIEW

**Comment by vanwagonet** [comment]
test plan run locally:
* `node scripts/generate-release-notes.js Student-6.8.1`
* `node scripts/generate-release-notes.js Student-6.7`
* Any other version you want to try out

Verify that the release notes are generated for that version. The 3rd line of output tells you what tags it used.

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/2afvcxu) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F2afvcxu)
teacher: [Link](https://rebrand.ly/mjwhcz0) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fmjwhcz0)
parent: [Link](https://rebrand.ly/pcw91ow) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fpcw91ow)

File: scripts/build_automation/automateVersioning.sh [hunk 1]
```diff
@@ -95,22 +95,9 @@ updateVersionAndBuildNumberInPlist() {
...
 	TAG="$APP_NAME-$APP_RELEASE_VERSION"
-	#	if the tag exists, use the tag 2nd to last in the list, else use the last tag in the list
-	NUMBER_FROM_THE_BOTTOM=1
-	if git ls-remote --tags | egrep -q "refs/tags/$TAG$"
-	then
-		NUMBER_FROM_THE_BOTTOM=2
-	fi
-	
-	
-	PREVIOUS_RELEASE_TAG=$(git ls-remote --tags origin | grep $APP_NAME- | sort -V | tail -$NUMBER_FROM_THE_BOTTOM | head -n 1 | awk '{print $2}' | sed -e "s/refs\/tags\///g")
```
Comment (vanwagonet) [inline scripts/build_automation/automateVersioning.sh:106]: the list of tags includes the hash first, so the `sort -V` was I think sorting by the hashes. I could have kept fixing here, but I like the generate-release-notes.js file just taking the current tag & figuring it out.

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

[MBL-14261](https://instructure.atlassian.net/browse/MBL-14261)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.6% | 93.6% | -0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**